src/map.cpp

```diff
@@ -649,32 +649,55 @@ const Tile* Map::canWalkTo(const Creature& creature, const Position& pos) const
 	return tile;
 }
 
-bool Map::getPathMatching(const Creature& creature, std::vector<Direction>& dirList,
+uint16_t calculateHeuristic(const Position& p1, const Position& p2)
```

Comment:
Is it a problem that this overflows for positions more than 256 tiles away from each other?

Reply:
How do you figure that?

Reply:
The code should only overflow on positions where: differenceX + differenceY > 65535

If you are trying to run large paths on this code you will need to change:

![image](https://github.com/user-attachments/assets/a2e857ff-2f60-4d27-a6ff-d2273c6e576f)

Reply:
Yeah no worries, just wanted to point that out. I didn't take a closer look into this, just wanted you to be aware of this.